### ISSUE TITLE
fix a npm audit security report

### ISSUE THREAD

**Issue by feelform** [pinpoint-apm/pinpoint-node-agent#21]
```
❯ npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
# Run  npm install find-node-modules@2.1.0  to resolve 1 vulnerability
SEMVER WARNING: Recommended action is a potentially breaking change
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ braces                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ find-node-modules                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ find-node-modules > findup-sync > micromatch > braces        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/786                             │
└───────────────┴──────────────────────────────────────────────────────────────┘


found 1 low severity vulnerability in 644 scanned packages
  1 vulnerability requires semver-major dependency updates.
```

**Comment by feelform** [comment]
```
$ nom install
28 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by feelform** [comment]
Fixed v0.7.2
